media-sound/gtick/gtick-0.5.4-r1.ebuild

```diff
@@ -0,0 +1,63 @@
+# Copyright 1999-2020 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+inherit xdg-utils
+
+DESCRIPTION="Metronome application supporting different meters and speeds ranging"
+HOMEPAGE="https://www.antcom.de/gtick"
+SRC_URI="https://www.antcom.de/gtick/download/${P}.tar.gz"
+
+LICENSE="GPL-3+"
+SLOT="0"
+KEYWORDS="~amd64 ~ppc ~sparc ~x86"
+IUSE="nls sndfile"
+
+RDEPEND="media-sound/pulseaudio
+	x11-libs/gtk+:2
+	sndfile? ( media-libs/libsndfile )"
+DEPEND="${RDEPEND}"
+BDEPEND="sys-devel/bison
+	sys-devel/flex
+	virtual/pkgconfig
+	nls? ( sys-devel/gettext )"
+
+RESTRICT="test"
+
+PATCHES=(
+	"${FILESDIR}"/0.5.4-fno-common.patch
+	"${FILESDIR}"/0.5.4-desktop.patch
+)
+
+src_prepare() {
+	default
+
+	sed -i 's:^\(appdatadir = .*/\)appdata:\1metainfo:' \
+		Makefile.in || die
+}
+
+src_configure() {
+	econf \
+		$(use_enable nls) \
+		$(use_with sndfile)
```

Comment:
To avoid these backslashes I convert to `local myeconfargs( ... )` array and `econf "${myeconfargs[@]}"` whenever I see those.